pkg/cli/style.go

```diff
@@ -2,19 +2,341 @@ package cli
+			TrueColor: "#2A0033",
+			ANSI256:   "55",
+			ANSI:      "4",
+		},
+	},
+}
+
+var colorViolet = Color{
+	Foreground: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#EE82EE",
+			ANSI256:   "13",
+			ANSI:      "5",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#EE82EE",
+			ANSI256:   "13",
+			ANSI:      "5",
+		},
+	},
+	Background: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#F5E6FF",
+			ANSI256:   "189",
+			ANSI:      "7",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#550055",
+			ANSI256:   "90",
+			ANSI:      "4",
+		},
+	},
+}
+
+var colorGray = Color{
+	Foreground: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#808080",
+			ANSI256:   "244",
+			ANSI:      "7",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#808080",
+			ANSI256:   "244",
+			ANSI:      "7",
+		},
+	},
+	Background: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#F2F2F2",
+			ANSI256:   "231",
+			ANSI:      "7",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#333333",
+			ANSI256:   "235",
+			ANSI:      "0",
+		},
+	},
+}
+
+var colorWhite = Color{
+	Foreground: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#FFFFFF",
+			ANSI256:   "15",
+			ANSI:      "7",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#FFFFFF",
+			ANSI256:   "15",
+			ANSI:      "7",
+		},
+	},
+	Background: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#F5F5F5",
+			ANSI256:   "231",
+			ANSI:      "7",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#333333",
+			ANSI256:   "235",
+			ANSI:      "0",
+		},
+	},
+}
+
+var colorBlack = Color{
+	Foreground: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#000000",
+			ANSI256:   "0",
+			ANSI:      "0",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#000000",
+			ANSI256:   "0",
+			ANSI:      "0",
+		},
+	},
+	Background: lipgloss.CompleteAdaptiveColor{
+		Light: lipgloss.CompleteColor{
+			TrueColor: "#E0E0E0",
+			ANSI256:   "248",
+			ANSI:      "7",
+		},
+		Dark: lipgloss.CompleteColor{
+			TrueColor: "#121212",
+			ANSI256:   "235",
+			ANSI:      "0",
+		},
+	},
+}
+
+////////
+
 var statusBarStyle = lipgloss.NewStyle().
-	Foreground(lipgloss.Color("#FFFDF5")).
-	Background(lipgloss.Color("#FF5F87")).
-	Padding(0, 1).
+	Padding(1).
 	MarginRight(1)
 
+var styleSuccessStatusBar = lipgloss.NewStyle().
+	Inherit(statusBarStyle).
+	Foreground(colorBlack.Foreground).
+	Background(colorGreen.Background).
+	Padding(0, 2).
+	MarginRight(1)
+
+var styleErrorStatusBar = lipgloss.NewStyle().
+	Inherit(statusBarStyle).
+	Foreground(colorBlack.Foreground).
+	Background(colorRed.Background).
+	Padding(0, 3).
+	MarginRight(1)
+
+var styleNoteStatusBar = lipgloss.NewStyle().
+	Inherit(statusBarStyle).
+	Foreground(colorYellow.Foreground).
```

Comment:
Just double-checking a foreground and background of the same color is intended here and in the `styleWarningStatusBar` directly below?

Reply:
I'm going to leave it for now since it looks decent.